components/KiboOrderSummary.vue

```diff
@@ -110,98 +72,112 @@ export default {
       type: Object,
       default: () => ({}),
     },
-    propertiesNames: {
-      type: Array,
-      default: () => ["Products", "Subtotal", "Shipping", "Total price"],
-    },
-    characteristics: {
-      type: Array,
-      default: () => [],
-    },
   },
-  setup(props) {
+  setup(props, context) {
     const { order } = props
 
-    const totalItems = computed(() => checkoutGetters.getLineItemTotal(order))
-    const shippingMethod = computed(() => checkoutGetters.getShippingMethod(order))
-    const subtotal = computed(() => checkoutGetters.getSubtotal(order))
-    const total = computed(() => checkoutGetters.getTotal(order))
+    const numberOfItems = computed(() => checkoutGetters.getLineItemTotal(order) || 0)
```

Comment:
can we move the || inside getter itself ?

Reply:
Resolved